src/EditorFeatures/CSharpTest2/Recommendations/DelegateKeywordRecommenderTests.cs

```diff
@@ -43,16 +43,25 @@ public async Task TestAfterGlobalVariableDeclaration_Interactive()
         }
 
         [Fact, Trait(Traits.Feature, Traits.Features.KeywordRecommending)]
-        public async Task TestNotInUsingAlias()
+        public async Task TestInUsingAlias()
```

Comment:
Test name still needs to be updated back to the original name?

Reply:
Fixed.